src/chains/index.ts

```diff
@@ -49,6 +51,14 @@ export const chainsLabel: Record<number, string> = backendNetworks.networks.redu
   {} as Record<number, string>
 );
 
+export const chainsPrivateMempoolTimeout: Record<number, number> = backendNetworks.networks.reduce(
+  (acc, backendNetwork: BackendNetwork) => {
+    acc[parseInt(backendNetwork.id, 10)] = backendNetwork.privateMempoolTimeout || DEFAULT_PRIVATE_MEMPOOL_TIMEOUT;
```

Comment:
once it exists in the backend, we will still need to update the graphql query to include it

Reply:
Can you create a ticket to do the arc related work?

Reply:
Placeholder here: https://linear.app/rainbow/issue/APP-2104/update-graphql-query-to-include-privatemempooltimeout-in-request